### ISSUE TITLE
Tooltips get right-to-left slide anim after mouse hovered over horde icon on edge of the window

### ISSUE THREAD

**Issue by Micovec** [Memori-Studios/CastlesOnClouds-Playtest#44]
### Current Behavior

All tooltips get right-to-left slide animation after hovering mouse over the horde icon when it's positioned on the right edge of the game's window.

The behaviour for tooltip on horde icon is correct (right-to-left) but all other tooltips such as those for powers get the same animation making some of them overflow the game's window and unreadable.

### Expected Behavior

Tooltips should have one unified animation except for those tooltips that would overflow the game's window.

### Steps To Reproduce

1. Position horde icon to right side of the game's window
2. Hover mouse over it, tooltip has right-to-left slide animation, that's correct
3. Hover mouse over any UI element that had previously left-to-right animation such as powers or middle bottom buttons
4. Tooltips have also right-to-left slide animation

### Playtest Version (center top)

v5.1.0

### Environment Specs

- OS: Arch Linux
- GPU: Nvidia RTX 3070
- CPU: Ryzen 5600X
- RAM: 32 GB

### Anything else?

To fix the right-to-left slide animation for other tooltips:

1. Position horde icon to the left side of the game's window
2. Hover mouse over it
3. Tooltips now have the original left-to-right slide animation

**Comment by Memori-Studios** [comment]
Yeah this tooltip is pretty broken due to updating constantly. Already spent too much time on it. Not a high enough priority atm but will revisit later in the polish process.
